### ISSUE TITLE
Event log with timestamps to be reminded of when things happened in SuperProductivity, and be able to assign time to tasks

### ISSUE THREAD

**Issue by leorochael** [johannesjo/super-productivity#3498]
## Problem Statement

Sometimes I'm working on a task where the time is being tracked in SuperProductivity when I'm interrupted by a colleague.

I pause the timer, so I can talk to the colleague without accruing the time to the original task (assume I cannot just ask the colleague to wait for the next Pomodoro pause), but if the discussion is extensive and work related, I'd like to capture this time to another task, new or existing.

Problem is, I might look at the clock and see what time is now, but I don't know what time it was when I clicked the pause button, so I can't calculate how much time I invested talking to my colleague.

The idle time detector from SuperProductivity couldn't help me in this case because the discussion with the colleague involved looking at the computer and navigating to different pages, so the computer wasn't idle.

## :grey_question: Possible Solution

It would be nice if I could at least go into an event log in the app that track all meaningful interactions, i.e. all interactions that caused the state to be changed in the app, but at the very least the starts and stops, whether automatic or manual, to look at the timestamp of when I clicked the pause button on a specific task.

## :arrow_heading_up: Describe alternatives you've considered

It would be even better if I could highlight periods of time within this log view to assign as time to a new or existing task, like the idle detector does.

**Comment by johannesjo** [comment]
Did you try using the tracking reminder? That should hopefully cover your case :)

**Comment by leorochael** [comment]
Not really, to be honest... I was afraid it was going to be disruptive, but I might give it a try.

Meanwhile I'm trying something else: I keep a "Discussion with Colleagues" task with a "Placeholder" subtask open.

Whenever a colleague comes to talk to me, I "play" the placeholder subtask, then when they're gone, I rename the subtask with the colleagues's name and subject.

I would still like the event log in any case, might give me insights in how I'm using the tool.
